src/processor/initializeSwap.ts

```diff
@@ -33,14 +44,61 @@ export async function initializeSwap(Data: {
     //     throw initializeData.warning;
     // }
     try {
-        const transactionHashs = await sendBundledTransactions({
-            provider: program.provider as AnchorProvider,
-            txsWithoutSigners: initializeData.txWithoutSigner,
-            signer: Data.signer,
-            clusterOrUrl: Data.clusterOrUrl,
-            simulation: Data.simulation,
-            skipConfirmation: Data.skipConfirmation,
-        });
+        let initTxs = initializeData.initTxs;
+
+        let transactionHashs: string[] = [];
+
+        if (initTxs) {
+            if (initTxs.init) {
+                await sendBundledTransactionsV2({
+                    txsWithoutSigners: initTxs.init,
+                    ...sendConfig,
+                })
+                    .then((txhs) => {
+                        transactionHashs.push(...txhs);
+                    })
+                    .catch((error) => {
+                        console.log("error", error);
+                    });
+            }
+
+            if (initTxs.add) {
```

Comment:
Maybe add a parameter:
if previous tx failed, we don't want to broadcast the rest
Now, we just catch error and log, no throw

Reply:
I removed catching, so there will be an error